### PR TITLE
feat(fe): redesign header and footer

### PR DESCRIPTION
**Pull request by Kimhyojung0810**
### Description

메인 페이지 헤더와 푸터 부분을 리디자인 했습니다.
스크린샷 2024-08-01 오후 7 43 14
스크린샷 2024-08-01 오후 7 43 18

### Additional context

---

### Before submitting the PR, please make sure you do the following

- [o] Read the [Contributing Guidelines](https://github.com/skkuding/next/blob/main/CONTRIBUTING.md)
- [o] Read the [Contributing Guidelines](https://github.com/skkuding/next/blob/main/CONTRIBUTING.md#pr-and-branch) and follow the [Commit Convention](https://github.com/skkuding/next/blob/main/CONTRIBUTING.md#commit-convention)
- [o] Provide a description in this PR that addresses **what** the PR is solving, or reference the issue that it solves (e.g. `fixes #123`).
- [ ] Ideally, include relevant tests that fail without this PR but pass with it.

### CODE REVIEW

File: apps/frontend/app/(main)/_components/Header.tsx [hunk 1]
```diff
@@ -8,18 +8,26 @@ import NavLink from './NavLink'
...
   return (
-    <header className="border-b-gray grid h-16 w-full place-items-center border-b bg-white px-5">
-      <div className="flex w-full max-w-7xl items-center justify-between gap-5">
+    <header className="fixed z-40 grid h-16 w-full place-items-center bg-white/80">
+      <div className="flex w-full max-w-7xl items-center justify-between gap-5 px-5">
         {/* FIXME: If you uncomment a group tab, you have to remove a pr-20 tailwind class */}
-        <div className="flex w-1/2 min-w-fit items-center justify-between gap-8 pr-20">
+        <div
+          className="flex min-w-fit items-center justify-between gap-8 text-[16px]"
+          style={{ width: '8.45325rem', height: '1.75rem' }}
```
Comment (jihorobert) [inline apps/frontend/app/(main)/_components/Header.tsx:16]: 이부분 없애도 될 것 같아요!!

File: apps/frontend/app/(main)/_components/Footer.tsx [hunk 2]
```diff
@@ -17,48 +19,49 @@ export default function Footer() {
...
   }
   return (
-    <footer className="mt-8 flex h-[100px] w-full items-center justify-center bg-gray-50 md:h-[125px] md:items-end">
-      <div className="flex w-full max-w-7xl flex-col justify-center gap-1 p-5 text-gray-400 md:flex-row md:justify-between md:gap-3">
-        <p className="text-center text-sm font-bold">
-          (c) SKKUDING / Since 2021
+    <footer className="mt-8 flex h-[380px] w-full flex-col items-center justify-center gap-8 bg-gray-50 md:h-[380px] md:items-center">
+      <div
+        className="flex min-w-fit items-center justify-center"
+        style={{ width: '12.07613rem', height: '2.5rem' }}
```
Comment (jihorobert) [inline apps/frontend/app/(main)/_components/Footer.tsx:25]: 이부분 없어도 될 것 같아요!!

File: apps/frontend/app/(main)/_components/Footer.tsx [hunk 3]
```diff
@@ -17,48 +19,49 @@ export default function Footer() {
     }
   }
   return (
-    <footer className="mt-8 flex h-[100px] w-full items-center justify-center bg-gray-50 md:h-[125px] md:items-end">
-      <div className="flex w-full max-w-7xl flex-col justify-center gap-1 p-5 text-gray-400 md:flex-row md:justify-between md:gap-3">
-        <p className="text-center text-sm font-bold">
-          (c) SKKUDING / Since 2021
+    <footer className="mt-8 flex h-[380px] w-full flex-col items-center justify-center gap-8 bg-gray-50 md:h-[380px] md:items-center">
```
[the remark this answers is not in the document]
Comment (Kimhyojung0810) [reply apps/frontend/app/(main)/_components/Footer.tsx:22]: 넵!! 반영했습니닷!!

File: apps/frontend/app/(main)/_components/Footer.tsx (hunk 2 above)
Comment (Kimhyojung0810) [reply apps/frontend/app/(main)/_components/Footer.tsx:25]: 완료 *ପ꒰⑅•ᴗ•｡꒱໊੭

File: apps/frontend/app/(main)/_components/Header.tsx (hunk 1 above)
Comment (Kimhyojung0810) [reply apps/frontend/app/(main)/_components/Header.tsx:16]: ヽ(•̀ω•́ )ゝ3개 다 수정했습니당 !!

**Review by jihorobert — APPROVED** [review]
수고하셨어요!

**Pull request merged**